clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1037,6 +1037,35 @@ public Void call() {
                 .getAccountToken() + " accountRegion: " + config.getAccountRegion());
     }
 
+    public void suspendInAppNotifications(){
```

Comment:
If we can write method descriptions for these public methods mentioning the role of `DISCARDED`, `RESUMED` and `SUSPENDED` states, we can clear the confusion especially  `SUSPENDED` and  `DISCARDED` states.

Reply:
Added Javadoc in latest commit